bin/merlin_file.ml

```diff
@@ -0,0 +1,56 @@
+open Stdune
+open Import
+
+let doc = "Generate merlin files (internal)"
+
+let man =
+  [ `S "DESCRIPTION"
+  ; `P
+      {| $(b, dune merlin-file) generates a .merlin file for a particular file.
+       This operation is internal to dune.
```

Comment:
It's not really internal if it's an interface for merlin, right?

Reply:
I just mean that it's not for end users and can change in a breaking way without any notice. Like the `format-file` command for example.